### ISSUE TITLE
Replay details crashes when there are no rrweb events

### ISSUE THREAD

**Issue by billyvg** [getsentry/sentry#33505]
This could be due to SDK issues, but we should display an error message when we have no rrweb events.

See https://sentry.io/organizations/sentry/issues/3119998388/?project=11276

**Comment by ryan953** [comment]
We're going to need better loading & error states in general.  <DetailedError> looks like where I'll start

**Comment by billyvg** [comment]
@ryan953 agreed! `<DetailedError>` might be a bit chonky but definitely good starting place.
